crates/loam-cli/tests/it/build_clients/init_script.rs

```diff
@@ -0,0 +1,42 @@
+use crate::util::TestEnv;
+
+#[test]
+fn build_command_runs_init() {
+    TestEnv::from("soroban-init-boilerplate", |env| {
+        env.set_environments_toml(
+            r#"
+development.accounts = [
+{ name = "alice" },
+{ name = "bob" },
+]
+
+[development.network]
+rpc-url = "http://localhost:8000/rpc"
+network-passphrase = "Standalone Network ; February 2017"
+
+[development.contracts.soroban_token_contract]
+workspace = true
+init = """
+STELLAR_ACCOUNT=bob initialize --symbol ABND --decimal 7 --name abundance --admin bob 
+STELLAR_ACCOUNT=bob mint --amount 2000000 --to bob 
```

Comment:
This is cool! I would also like to see a test without `STELLAR_ACCOUNT`. The `default_account` logic in `build_clients` should mean it works even without this. You're using an implicit default here, so you'll probably need to set `--to alice`, since she's the default.

Reply:
Sure I'll add another test that doesnt use the STELLAR_ACCOUNT directive (I was kind of just combining two tests into one here)